2-mandatory/4-stocks.js

```diff
@@ -33,9 +33,24 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Solve the smaller problems, and then build those solutions back up to solve the larger problem.
         Functions can help with this!
 */
+
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    // let arr = [];
+    let averagePrices = closingPricesForAllStocks
+        .map((el) => {
+            // console.log(el);
+            return (
+                el.reduce((acc, value, _, { length }) => {
+                    // console.log(length);
+                    return acc + value;
+                }) / el.length
```

Comment:
Good use of reduce - this is advanced knowledge. I'm not sure what "_, { length } "does here - the code seems to work without it. You can explain it to me when we meet next week.